src/main/java/net/aufdemrand/denizen/events/entity/FireworkBurstsScriptEvent.java

```diff
@@ -90,6 +96,9 @@ else if (name.equals("location")) {
     public void onFireworkBursts(FireworkExplodeEvent event) {
         entity = new dEntity(event.getEntity());
         location = new dLocation(entity.getLocation());
+        ItemStack itemStack = new ItemStack(Material.FIREWORK);
+        itemStack.setItemMeta(event.getEntity().getFireworkMeta());
+        item = new dItem(itemStack);

```

Comment:
...so not only is this not necessary at all, since you can just do `<context.entity.firework_item>`, but you didn't put the creation of the item in the right place or even actually add the `<context.item>` tag